lib/Elastica/Aggregation/DateHistogram.php

```diff
@@ -11,7 +12,7 @@ class DateHistogram extends Histogram
     /**
      * Set pre-rounding based on interval.
      *
-     * @deprecated Option "pre_zone" is deprecated as of ES 1.5. Use "time_zone" instead
+     * @deprecated Option "pre_zone" is deprecated as of ES 1.5 and will be removed in Elastica 4.0. Use "time_zone" instead
```

Comment:
I would probably not name a "specific" version of Elastica, as I'm also willing to remove deprecated functions in minor version.